clang/include/clang/Basic/CodeGenOptions.h

```diff
@@ -64,6 +64,12 @@ class CodeGenOptions : public CodeGenOptionsBase {
     Darwin_libsystem_m // Use Darwin's libsytem_m vector functions.
   };
 
+  enum AltMathLibrary {
+    NoAltMathLibrary,   // Don't use any alternate math library
+    SVMLAltMathLibrary, // INTEL SVML Library
```

Comment:
Intel here should not be in all caps.

Reply:
Hi Andy, given that introducing SVML support for fp builtin selection here doesn't help because it will introduce linker errors. Do you think we need to support it now?